webcompat/static/js/lib/wizard/utils.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const HEADER_HEIGHT = 130;
+
+const utils = {
+  scrollToElement: (element, delay = 250) => {
+    // Delay "scroll to element" effect in order to let the animation finish, otherwise the scroll point isn't correct
+    setTimeout(() => {
+      const topOfElement =
+        window.pageYOffset +
+        element.getBoundingClientRect().top -
+        HEADER_HEIGHT;
+      window.scroll({ top: topOfElement, behavior: "smooth" });
```

Comment:
Do we know what happens in Safari? It looks like maybe `ScrollToOptions` is not supported?

https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll

Reply:
It's jumpy :| This was also moved from the existing script. I can add a separate issue for that

Reply:
Cool, thanks. We're bug-compatible! 😄